assignment.js

```diff
@@ -31,6 +31,10 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+let baseNumber = 1
+for (let i = 0; i < 13; i++) {
+  console.log(baseNumber * i)
+}
 
```

Comment:
@laraann88 Good job using the for loop to complete the task. I would recommend when you're using a loop instead of using `i < 13` try `i <= 12`. take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L38)